### ISSUE TITLE
Errors when launching pyportify

### ISSUE THREAD

**Issue by emilime93** [rckclmbr/pyportify#32]
Hi!

After finally being able to install everything without errors i get an error when launching pyportify. I get a similar error when trying to launch pyportify-copyall aswell. This is the error message from simply launching "pyportify":

Traceback (most recent call last):
  File "/usr/local/bin/pyportify", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/Library/Python/2.7/site-packages/distribute-0.6.14-py2.7.egg/pkg_resources.py", line 2671, in <module>
    working_set.require(__requires__)
  File "/Library/Python/2.7/site-packages/distribute-0.6.14-py2.7.egg/pkg_resources.py", line 654, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Library/Python/2.7/site-packages/distribute-0.6.14-py2.7.egg/pkg_resources.py", line 552, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: six>=1.6.1

Mac OSX 10.10.3
Help is greatly appreciated!

**Comment by rckclmbr** [comment]
Try installing the brew python distribution

```
brew install python
```

See https://github.com/rckclmbr/pyportify/issues/20 and try that

**Comment by emilime93** [comment]
Thanks for responding!
I already installed python through brew, though I had a previous installation of Python that I removed (and reinstalled brew verison). Now I get another error:

$ pyportify
-bash: /usr/local/bin/pyportify: /usr/bin/python: bad interpreter: No such file or directory

Sorry if I'm asking blatant questions but I'm not used to doing these kind of things. I only have basic Unix/terminal knowledge. Installed both brew and pip for the first time when following your install instructions.
